### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Ryad Bouzourène, Anthony Christen, Louis Haye

### DISCUSSION THREAD

**Discussion by Artotrogo** [heig-vd-dai-course/heig-vd-dai-course#555] [Show and tell]
Notre API, qui sera relié notre projet du cours de BDR, aura pour but de faire des requêtes vers une base de données concernant le suivi diététique de patients d'un hôpital. 
ID github : 
- Ryad : ryadbouzourene 
- Anthony : Anthony-Christen

Cahier des charges (non màj) :
[cahier_des_charges.pdf](https://github.com/user-attachments/files/18208749/cahier_des_charges.pdf)

**Comment by ludelafo** [comment]
Comme discuté, c'est OK pour moi mais je n'ai pas vu passer de mail avec Juergen pour confirmer que c'est OK. Merci !
